rune/libenclave/attestation/internal/sgx/ias/ias.go

```diff
@@ -61,9 +57,9 @@ func NewIasAttestation(cfg map[string]string) (*IasAttestation, error) {
 		return nil, fmt.Errorf("EPID parameter subscription-key not specified")
 	}
 
-	if len(subKey) != subscriptionKeyLength*2 {
+	if len(subKey) != intelsgx.SubscriptionKeyLength*2 {
 		return nil, fmt.Errorf("Subscription key must be %d-character long",
-			subscriptionKeyLength*2)
+			intelsgx.SubscriptionKeyLength*2)
```

Comment:
ecdsa also uses this variable, so it is necessary to rename it as `AttestationSubscriptionKeyLength`.

Reply:
> ecdsa also uses this variable, so it is necessary to rename it as `AttestationSubscriptionKeyLength`.

Done, thanks.